CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Platform Version 0.11.6 - 2024-04-20
+
+### Changed
+
+* Released `v0.11.6-dev-1`
```

Comment:
Can you clean up the Changelog.  Remove the 0.11.6-dev header and merge the "* Released..." line so it's all under `## Platform Version 0.11.6 - 2024-04-20`

Reply:
Of course! Here is the desired change applied: https://github.com/infinyon/fluvio/pull/3954/commits/93858fce6521cbf7f4e30bf33528ec4d70b60a08